config/aws_citc.py

```diff
@@ -19,6 +19,14 @@
 # Override with RFM_PREFIX environment variable
 reframe_prefix = os.path.join(os.environ['HOME'], 'reframe_runs')
 
+# This enables runtime selection of which EESSI_VERSION should be loaded. This is used e.g. in CI.
+eessi_version = os.getenv('EESSI_VERSION', 'latest')
+if eessi_version == 'latest':
+    eessi_init_script = '/cvmfs/pilot.eessi-hpc.org/latest/init/bash'
+else:
+    eessi_init_script = '/cvmfs/pilot.eessi-hpc.org/versions/%s/init/bash' % eessi_version
```

Comment:
@casparvl Can we avoid copy-pasting this exact same piece of code, and define `eessi_init_script` in `common_config` instead and `import` it from there?

Reply:
Done